.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "airbnb",
+    "plugins": [
+        "react"
```

Comment:
Whoops, good catch. I had it for some testing stuff but it's not necessary. Removed.